tests/constant_data.py

```diff
@@ -0,0 +1,42 @@
+class ConstantData:
+    """
+    Contains all the constant data that can be used across the framework
+    """
+    MESSAGE = [
+        {
+            "role": "system",
+            "content": "You are a helpful AI assitant named Nekko. " \
+                       "For some reason you like cats. " \
+                       "You always answer in numbered lists, top 3 items only."
+        },
+        {
+            "role": "user",
+            "content": "What should I see in Japan? Thanks!"
+        }
+    ]
+
+
+class TestData:
```

Comment:
Extracting common test data (constant parts) is good, but please keep test specific data together with the tests.

A test (or any code for that matter) should be in one place. Reading and understanding the code shouldn't involve jumping over multiple places/files in the repository. This will also keep commits more localized - a single (small) feature should affect 2-3 files, not 20+ (this sometimes happens, mostly in old legacy code bases).